docs/cpp/source/installing.rst

```diff
@@ -154,6 +154,71 @@ should now merrily print the tensor (exact output subject to randomness):
   Also, make sure you specify the correct configuration in the ``cmake --build .``
   line above.
 
+System Requirements
+-------------------
+
+To ensure smooth installation and usage of LibTorch, please ensure your system
+meets the following requirements:
+
+1. **GLIBC Version**:
+  - GLIBC 2.29 or newer for cxx11 ABI version
+  - GLIBC 2.17 or newer for pre-cxx11 ABI version
+
+
+Troubleshooting
+---------------
+
+To check the dependencies of the LibTorch libraries and identify the required
+GLIBC version, you can use the following command from the unzip directory:
+
+.. code-block:: sh
+
+  ldd lib/libtorch.so
+
+If your system do not have the required GLIBC version, the output will look
+like this:
```

Comment:
while I appreciate the idea here, I'm not sure that this gives the user much more information than they'd get from just trying to use libtorch. I realize that ensuring the correct GLIBC version is installed is very OS / environment specific, so it's not really feasible to solve that problem here.

Reply:
Makes sense, do you think is best to remove the whole troubleshooting section then?

Reply:
I think so yes